packages/webapp/pages/account/profile.tsx

```diff
@@ -124,15 +125,25 @@ const AccountProfilePage = (): ReactElement => {
               id={coverId}
               className={{
                 root: 'absolute left-0 top-0 flex w-full',
-                container: 'border-0 bg-theme-bg-secondary',
+                container:
+                  'border-0 bg-theme-bg-secondary hover:bg-theme-bg-tertiary',
                 img: 'object-cover',
               }}
               size="cover"
               initialValue={user.cover}
               fallbackImage={null}
-              hoverIcon={<CameraIcon size={IconSize.Large} />}
+              alwaysShowHover={!user.cover && !coverImage}
+              hoverIcon={
+                <span className="ml-26 mr-3 flex flex-wrap items-center justify-center text-theme-label-secondary">
```

Comment:
Unfortunately the layout is not so simple. The button element is displayed for the whole width. The profile picture is on the left side and is displayed over the button for the cover image. Profile picture is displayed with border radius and the cover image slips into the gaps that the border radius leaves visible of the cover image button. That's why we cannot make the cover button smaller and also why adding padding to the cover button itself would not work.

So given that the profile image button takes part of the element width, the easiest way to actually center the label is using the increased left padding on the label.

A couple of screenshots to better illustrate what I mean:
<img width="406" alt="Screenshot 2024-01-25 at 20 57 40" src="https://github.com/dailydotdev/apps/assets/9974711/1c12d28c-db24-415d-b362-8667fec4dde6">
<img width="409" alt="Screenshot 2024-01-25 at 20 58 02" src="https://github.com/dailydotdev/apps/assets/9974711/65d3bf37-343c-43e3-9804-1347edc61faa">
<img width="410" alt="Screenshot 2024-01-25 at 20 57 49" src="https://github.com/dailydotdev/apps/assets/9974711/b2e848e0-e623-418b-81d4-e5e38864c6e7">

Reply:
Maybe rework the layout, instead of having absolute placement, we render it as a regular flex row. Probably to make rendering the span easier. Just not entirely sure if it is actually easier with the rework 😂

Reply:
Yeah I was also thinking:

<div> // background
<profile pic><rest span upload cover>
</div>

Reply:
Good idea, I'll take a look 👍